docs/adding-a-new-lgsl-code.md

```diff
@@ -18,4 +26,20 @@ Providing tiers can be:
 - `county/unitary`
 - `district/unitary`
 
-Once your pull request has been merged the codes will become active overnight.
+## Activate relevant service interactions
```

Comment:
You started with `1.`, `2.` in your headings above but they seem to be missing from here onwards, should they be in these ones too?

Reply:
Updated.